### ISSUE TITLE
Vein Miner adds Mending - Bug

### ISSUE THREAD

**Issue by Puffball920** [Iskallia/Vault-public#440]
**Describe the bug**
Sometimes it can happen, that Vein Miner glitches out, so whatever tool you're using in your Off-hand will start to mend. As long as either one of your tools have mending as enchants on them, you will be able to mend anything that's mendable. The blocks you're breaking do not need to drop xp at all. The other tool doesn't need any enchants at all either.

**To Reproduce**
1. Equip one tool with mending on either main- or offhand
2. Equip other tool/ weapon/ armor peace on the other
3. Mine literally anything with veinminer on
4. Thing get mended suddenly

**Expected behavior**
I didn't expect this to happen, as I was just trying to break normal boring stone with my pickaxe, my unenchanted axe in my off-hand

**Screenshots or Crash Report**
Not sure how I can show this with screenshots.

**Info (please complete the following information):**
- World Type: Iskallium VH Server
- The_Vault Version: 1.5

**Comment by jmilthedude** [comment]
Hello, I am unable to replicate this. I will close this unless you can potentially record this happening. I tried the following:
- equip iron pickaxe with mending.
- equip iron axe in off hand
- mine stone, nothing mended.
- mine coal and collect xp, mending pickaxe mended as expected, iron axe did not mend.

**Comment by Puffball920** [comment]
oh.. it got closed before I was able to record it.. I just got it again
too, I would've recorded my screen with obs, but since it's closed, do I
not do it anymore?


Am Mo., 11. Okt. 2021 um 15:08 Uhr schrieb Justin ***@***.***

**Comment by Puffball920** [comment]
I figured out why it started to do it, it happens when you have a food item in the off-hand and start eating it.
It seems to just switch enchants for some reason, it doesn't seem to mend the tools anymore, but it still uses durability.

> Hello, I am unable to replicate this. I will close this unless you can potentially record this happening. I tried the following:
> 
> * equip iron pickaxe with mending.
> * equip iron axe in off hand
> * mine stone, nothing mended.
> * mine coal and collect xp, mending pickaxe mended as expected, iron axe did not mend.

I figured out why it started to do it, it happens when you have a food item in the off-hand and start eating it.
It seems to just switch enchants for some reason, it doesn't seem to mend the tools anymore, but it still uses durability.
